frappe/public/js/frappe/form/print.js

```diff
@@ -190,6 +212,49 @@ frappe.ui.form.PrintPreview = Class.extend({
 						callback: function (data) {
 						}
 					});
+				} else if (me.get_mapped_printer().length === 1) {
+					// printer is already mapped in localstorage (applies for both raw and pdf )
+					if (me.is_raw_printing()) {
+						me.get_raw_commands(function (out) {
+							let printer_map = me.get_mapped_printer()[0];
+							let config = qz.configs.create(printer_map.printer);
+							let data = [out.raw_commands];
+							frappe.ui.form.qz_connect().then(function () {
+								return qz.print(config, data);
+							}).then(frappe.ui.form.qz_success).catch((err) => {
+								frappe.ui.form.qz_fail(err);
+							});
+						});
+					} else {
+						frappe.show_alert({
+							message: __('PDF printing via "Raw Print" is not yet supported. Please remove the printer mapping in Raw Printing Settings and try again.'),
+							indicator: 'blue'
+						}, 14);
+						//Note: need to solve "Error: Cannot parse (FILE)<URL> as a PDF file" to enable qz pdf printing.
+
+						// // use pdf method print method of qz
+						// let printer_map = me.get_mapped_printer()[0]
```

Comment:
remove commented code

Reply:
thought it would make it easier to enable PDF printing via QZ Tray after solving the Error... 

I have removed it now.